pulsar-broker/src/main/java/com/yahoo/pulsar/broker/lookup/LookupResult.java

```diff
@@ -30,41 +30,41 @@
  */
 public class LookupResult {
     enum Type {
-        BrokerUrl, HttpRedirectUrl
+        BrokerUrl, RedirectUrl
     }
 
     private final Type type;
     private final LookupData lookupData;
-    private final URI httpRedirectAddress;
 
     public LookupResult(NamespaceEphemeralData namespaceEphemeralData) {
         this.type = Type.BrokerUrl;
-        this.lookupData = new LookupData(namespaceEphemeralData.getNativeUrl(),
-                namespaceEphemeralData.getNativeUrlTls(), namespaceEphemeralData.getHttpUrl());
-        this.httpRedirectAddress = null;
+		this.lookupData = new LookupData(namespaceEphemeralData.getNativeUrl(),
+				namespaceEphemeralData.getNativeUrlTls(), namespaceEphemeralData.getHttpUrl(),
+				namespaceEphemeralData.getHttpUrlTls());
     }
 
-    public LookupResult(URI httpRedirectAddress) {
-        this.type = Type.HttpRedirectUrl;
-        this.lookupData = null;
-        this.httpRedirectAddress = httpRedirectAddress;
-    }
+	public LookupResult(String httpUrl, String httpUrlTls, String brokerServiceUrl, String brokerServiceUrlTls) {
```

Comment:
The indent width seems different. I guess it's tab vs space issue.

We should format codes before committing since we have format rules in `src/formatter.xml`.